apps/replication_tunable_consistency/cluster_healthy.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"log"
+	"os"
+	"time"
+
+	"github.com/weaviate/weaviate-go-client/v4/weaviate/data/replication"
+	"github.com/weaviate/weaviate/entities/models"
+)
+
+func main() {
+	ctx, cancel := context.WithTimeout(context.Background(), time.Hour)
+	defer cancel()
+
+	log.Printf("Validate all objects were added, with consistency level ONE...")
+
+	b, err := os.ReadFile("data/data.json")
```

Comment:
Maybe we can refactor reading the objects into a function and reuse it the other files

Reply:
Good idea, done